packages/flutter/lib/src/widgets/icon_data.dart

```diff
@@ -66,3 +66,43 @@ class IconData {
   @override
   String toString() => 'IconData(U+${codePoint.toRadixString(16).toUpperCase().padLeft(5, '0')})';
 }
+
+/// [DiagnosticsProperty] that has an [IconData] as value.
+class IconDataDiagnosticsProperty extends DiagnosticsProperty<IconData> {
+  /// Create a diagnostics property for strings.
```

Comment:
This comment is wrong.
Also consider whether all these named parameters make sense for an Icon. I would expect almost all of them are dead code from copying the implementation for StringProperty.

Reply:
Updated comment and removed all currently unused properties.